### ISSUE TITLE
Softbans: unbanning doesn't remove restrictions

### ISSUE THREAD

**Issue by dsent** [umputun/tg-spam#149]
When the bot is in the softban mode, clicking on "change ban" and unbanning a user wrongly detected as spammer, doesn't remove the restriction enacted by the bot.
Workaround: manually remove restrictions set by the bot after unbanning.

**Comment by dsent** [comment]
Okay, it seems that the bot does remove some restrictions, but the restriction entries remain in the list.
Next time I unban someone, I'll try to see what restrictions in particular remain active after unban.
![image](https://github.com/user-attachments/assets/e9e292ec-ede2-4c22-83f4-8e0651adf5b8)

**Comment by dsent** [comment]
Okay, a bit more details.
Here is the original restriction change done by the bot when I press "unban":
![image](https://github.com/user-attachments/assets/7ba73810-f4da-4065-86dd-d9b15ed9ec4d)
And here is the change that happened when I removed the remaining entry from the list of restrictions:
![image](https://github.com/user-attachments/assets/f01365fc-6be8-44ad-b24d-fadac8ad0be6)

**Comment by umputun** [comment]
Good catch. There was a mismatch between banned and unbanned permissions; they should be identical now. In a few moments, as the matter is built, you can give it a try.

**Comment by umputun** [comment]
released with [v1.14.0](https://github.com/umputun/tg-spam/releases/tag/v1.14.0)
